lib/ansible/modules/cloud/vmware/vmware_host_facts.py

```diff
@@ -277,6 +279,8 @@ def get_system_facts(self):
             'ansible_product_serial': sn,
             'ansible_bios_date': self.host.hardware.biosInfo.releaseDate,
             'ansible_bios_version': self.host.hardware.biosInfo.biosVersion,
+            'ansible_uptime': self.host.summary.quickStats.uptime,
```

Comment:
Could you please create a separate PR for this change ? Since these two changes are not related  to each other.

Reply:
Sounds good